src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -239,463 +245,66 @@ private Object createObject(String className) throws CheckstyleException {
     }
 
     /**
-     * Fill short-to-full module names map.
-     */
-    private static void fillShortToFullModuleNamesMap() {
```

Comment:
I don't believe we are expecting these hardcoded lists to be removed, but we are still waiting more word from @romani .